tasks/src/zero-shot-image-classification/about.md

```diff
@@ -8,4 +8,23 @@ The data in this learning paradigm consists of
 
 - Seen data - images and their corresponding labels
 - Unseen data -  only labels and no images
-- auxiliary information - additional information given to the model during training connecting the unseen and seen data. This can be in the form of textual description or word embeddings.
\ No newline at end of file
+- auxiliary information - additional information given to the model during training connecting the unseen and seen data. This can be in the form of textual description or word embeddings.
+
+
+## Use Cases
+
+### Autonomous Vehicles
+Autonomous Vehicles are a perfect use case for Zero-shot image classification. Occasionally in real-life situations, even we humans encounter new objects on the road that we are not sure how to react to instantly. Detecting + Classifying novel objects and deciding the correct actions to take are crucial for Robust Autonomous Vehicle Systems.
+
+### Image Retrieval
+Zero-shot learning resolves several challenges in existing Image retrieval systems. For example, with the rapid growth of unseen categories on the web, it is challenging to index images based on known categories.
+
+Zero-shot learning also has applicability in Image Instance Retrieval, where against an input image, images representing the same features are retrieved from a database. Conventional systems can only search for  semantically-related images, not always the exact instances.
```

Comment:
Can you express more about the intuition behind this use case and how it resolves? (above you only wrote about problems and then continued with different applications)

Reply:
ive added a low level detail behind the intuition, i can elobrate futher if need be.

Reply:
> Zero-shot learning also has applicability in Image Instance Retrieval, where against an input image, images representing the same features are retrieved from a database. Conventional systems can only search for  semantically-related images, not always the exact instances.

Maybe add a one-liner to let the readers know how exactly that's done (with the use of descriptions)?

Reply:
Ive decided to remove the second paragraph. I'm unable to access the complete research paper to write a proper description. Also, if the use cases seem to be too short-listed now I can work towards adding some more.

Reply:
SGTM.